qa/tasks/cephfs/test_scrub_checks.py

```diff
@@ -11,6 +11,108 @@
 
 log = logging.getLogger(__name__)
 
+class TestScrubControls(CephFSTestCase):
+    """
+    Test basic scrub control operations such as abort, pause and resume.
+    """
+
+    MDSS_REQUIRED = 1
+    CLIENTS_REQUIRED = 1
+
+    def _abort_scrub(self, expected):
+        mds_id = self.fs.get_lone_mds_id()
+        res = json.loads(self.fs.mon_manager.raw_cluster_cmd("tell", "mds.{0}".format(mds_id), "scrub", "abort"))
```

Comment:
Let's define a Filesystem method that gets rank 0's id and executes ceph tell with the given command.